src/style.css

```diff
@@ -1,3 +1,125 @@
-body {
+/* #region CSS Reset */
+html,
+body,
+div,
+span,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+a,
+em,
+img,
+footer,
+header,
+nav,
+ul,
+ol,
+li {
   margin: 0;
 }
+
+/* #endregion */
+
+/* #region Page  */
+.html {
+  font-family: Roboto, sans-serif;
+
+  --header-height: 60px;
+  --moyo-blue-color: #00acdc;
+}
+
+
+/* #endregion */
+
+/* #region Fonts  */
+.roboto-regular {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+.roboto-bold {
+  font-family: Roboto, sans-serif;
+  font-weight: 700;
+  font-style: normal;
+}
+
+.roboto-regular-italic {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: italic;
+}
+
+.roboto-bold-italic {
+  font-family: Roboto, sans-serif;
+  font-weight: 700;
+  font-style: italic;
+}
+
+/* #endregion */
+
+/* #region Header */
+.header {
+  height: var(--header-height);
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.header__logo_url {
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  display: flex;
+  background-color: #fff;
+  height: var(--header-height);
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+}
+
+.nav__url {
+  display: block;
+  height: var(--header-height);
+  margin: 0 10px;
+
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: var(--header-height);
+}
+
+.nav__url:hover {
+  color: var(--moyo-blue-color);
+}
+
+.is-active {
+  color: var(--moyo-blue-color);
+}
+
+.is-active::after {
+  content: '';
+  display: block;
+  background-color: var(--moyo-blue-color);
+  height: 4px;
+  width: 37px;
```

Comment:
```suggestion
  width: 100%;
```

Reply:
Almsot broke my head, thinking how to make it universal length, which will fit any element, without .js
I was so easy 🫣🤦
Thanks